src/client.ts

```diff
@@ -2744,14 +2744,14 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
    * scheduleCampaign - Schedule a Campaign
    *
    * @param {string} id Campaign ID
-   * @param {{sendAt: number}} params Schedule params
+   * @param {{scheduledFor: number}} params Schedule params
```

Comment:
does it have the same meaning as before?

Reply:
yes, a new field is introduced on the API side which reflects this field.

Reply:
isn't it less consistent to use `_for` suffix instead of `_at`, compared to other endpoints?

Reply:
on campaign side we have two fields now:
- scheduled_at: at which time the campaign was scheduled
- scheduled_for: for when the campaign is scheduled (sendAt here)